components/server/src/jobs/installation-admin-cleanup.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { BUILTIN_INSTLLATION_ADMIN_USER_ID, UserDB } from "@gitpod/gitpod-db/lib";
+import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
+import { inject, injectable } from "inversify";
+import { Job } from "./runner";
+
+@injectable()
+export class InstallationAdminCleanup implements Job {
+    @inject(UserDB) protected readonly userDb: UserDB;
+
+    public name = "installation-admin-cleanup";
+    public frequencyMs = 5 * 60 * 1000; // every 5 minutes
```

Comment:
If would be sufficient to execute this only once, but I think this routine isn't expensive.